README.md

```diff
@@ -35,6 +39,22 @@ const serialized = serialize({any: 'serializable'});
 const deserialized = deserialize(serialized);
 ```
 
+CJS example:
+
+```js
+// as single funcion
+const {default: structuredClone} = require('@ungap/structured-clone');
```

Comment:
this is needed only if code is not transformed, as Babel would do the right thing there without needing to explicitly access the `default` ... maybe worth mentioning?